test/parser.jl

```diff
@@ -83,3 +83,14 @@ macro test999_str(args...); args; end
 
 # issue #10994
 @test parse("1 + #= \0 =# 2") == :(1 + 2)
+
+# issue #10910
+@test parse(":(using A)") == :(:(using A))
```

Comment:
On May 22, 2015 12:00 PM, "Jeff Bezanson" <notifications@github.com> wrote:
>
> In test/parser.jl:
>
> > @@ -83,3 +83,14 @@ macro test999_str(args...); args; end
> >
> >  # issue #10994
> >  @test parse("1 + #= \0 =# 2") == :(1 + 2)
> > +
> > +# issue #10910
> > +@test parse(":(using A)") == :(:(using A))
>
> I'm not sure this really counts as a test of the parser; it only tests
that the parse function does the same thing as the parser used to read
source files. You have to examine the parsed expression in some other way.

I see your point . The original goal is to just make sure it doesn't raise
an error. I guess making the @test actually t3DT sth is better.

>
> —
> Reply to this email directly or view it on GitHub.